find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
+  max = nil
+  if max_index == 0
+    puts "Array is empty"
+  else
+    max = array[0]
+    index = 0
```

Comment:
since you already set the max to the value of the first indexed item, you can set the initial index to 1 so that you're not comparing the value of the first index to itself.